examples/apps/contact-collection/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluid-example/contact-collection",
-  "version": "0.58.0",
+  "version": "0.58.1000",
```

Comment:
is next ready for changes right now? do people know that main can't take breaks starting when the 0.58 branch was cut?

Reply:
next has policy/etc applied and can take changes.  it will still need to take this change then bump its version after this goes in though, so not quite fully ready.  i'll send out the warning email/update on policies soon